docs/sphinx/user_guide/feature/local_array.rst

```diff
@@ -40,59 +40,83 @@ in which we construct and use two arrays in nested loops::
 
            }
 
-In the example above, we see that two arrays have been constructed inside the 
-k - for loop and used in both j - for loops. This loop pattern may be expressed
-similarly using RAJA local arrays in RAJA kernel. Next, we show the RAJA variant of
-these for loops and discuss the different components:: 
+Here, two stack-allocated arrays are defined inside the outer 'k' loop and 
+used in both inner 'j' loops. This loop pattern may be also be expressed 
+using RAJA local arrays in a ``RAJA::kernel_param`` kernel. We show a 
+RAJA variant below, which matches the implementation above, and then discuss 
+its constituent parts::
+
+  // 
+  // Define two local arrays
+  // 
 
   using RAJA_a_array = RAJA::LocalArray<int, RAJA::Perm<0, 1>, RAJA::SizeList<5,7> >;
   RAJA_a_array kernel_a_array;
 
   using RAJA_b_array = RAJA::LocalArray<int, RAJA::Perm<0>, RAJA::SizeList<5> >;
   RAJA_b_array kernel_b_array;
 
-  using Pol =  RAJA::KernelPolicy<
-                 RAJA::statement::For<1, RAJA::loop_exec,
-                   RAJA::statement::InitLocalMem<RAJA::cpu_tile_mem, RAJA::ParamList<0, 1>,
-                     RAJA::statement::For<0, RAJA::loop_exec,
-                       RAJA::statement::Lambda<0>
-                     >,
+
+  // 
+  // Define the kernel execution policy
+  // 
+
+  using POL = RAJA::KernelPolicy<
+                RAJA::statement::For<1, RAJA::loop_exec,
+                  RAJA::statement::InitLocalMem<RAJA::cpu_tile_mem, RAJA::ParamList<0, 1>,
+                    RAJA::statement::For<0, RAJA::loop_exec,
+                      RAJA::statement::Lambda<0>
+                    >,
                     RAJA::statement::For<0, RAJA::loop_exec,
                       RAJA::statement::Lambda<1>
                     >
                   >
                 >
               >;
 
-  RAJA::kernel_param<Pol> (
-  RAJA::make_tuple(RAJA::RangeSegment(0,5), RAJA::RangeSegment(0,7)),
-  RAJA::make_tuple(kernel_a_array, kernel_b_array),
 
-    [=] (int j, int k, RAJA_a_array &kernel_a_array, RAJA_b_array &kernel_b_array) {
-      kernel_a_array(k, j) = 5*k + j;
-      kernel_b_array(j) = 5*k + j;
+  // 
+  // Define the kernel
+  // 
+
+  RAJA::kernel_param<POL> ( RAJA::make_tuple(RAJA::RangeSegment(0,5), 
+                                             RAJA::RangeSegment(0,7)),
+                            RAJA::make_tuple(kernel_a_array, kernel_b_array),
+
+    [=] (int j, int k, RAJA_a_array& a_array, RAJA_b_array& b_array) {
+      a_array(k, j) = 5*k + j;
+      b_array(j) = 5*k + j;
     },
 
-    [=] (int j, int k, RAJA_a_array &kernel_a_array, RAJA_b_array &kernel_b_array) {
+    [=] (int j, int k, RAJA_a_array& a_array, RAJA_b_array& b_array) {

```

Comment:
kernel_a_array and kernel_b_array were converted to a_array and b_array here, but not in other parts of the file. Converting them to a_array and b_array might cause some confusion with the native C version.

Reply:
Changed back.